pages/common/virt-sparsify.md

```diff
@@ -0,0 +1,13 @@
+# virt-sparsify
+
+> libguestfs tool to to make a virtual machine drive images thin-provisioned.
+> Use only for offline machines to avoid data corruption.
+> Home page: <https://libguestfs.org/>.
+
+- Create sparsified comporessed image without snapshots from unsparsified:
+
+`virt-sparsify --compress {{image.qcow2}} {{image_new.qcow2}}`
+
+- Sparsify image in place:
+
+`virt-sparsify --in-place {{/var/lib/libvirt/images/image.img}}`
```

Comment:
Consistency again
```suggestion
`virt-sparsify --in-place {{path/to/image.img}}`
```
....unless images _have_ to be located in the directory you've specified there in order for it to work properly - in which case ignore this comment.